README.md

```diff
@@ -7,28 +7,65 @@
 This package is part of the [Symfony Content Management Framework (CMF)](http://cmf.symfony.com/)
 and licensed under the [MIT License](LICENSE).
 
-The Slugifier API provides a basic interface to use for third party slugifiers.
-Besides this, it comes with a usefull `CallbackSlugifier` which is capable of
-handling most third party slugifiers.
+A "slugifier" is a function which transforms a string `such as this` into a
+URL-friendly string `such-as-this`. Slugifiers are also known as "urlizers".
 
+This package does not contain a slugifier itself, but provides a standard
+interface to use for third party slugifiers.  Besides this, it comes with a
+useful `CallbackSlugifier` which is capable of handling most third party
+slugifiers.
```

Comment:
which is capable of wrapping most third party slugifiers to the SlugifierInterface provided by this package.